codegen.mts

```diff
@@ -8,15 +8,7 @@ const config: CodegenConfig = {
     // API
     './packages/services/api/src/modules': {
       preset: 'graphql-modules',
-      plugins: [
-        {
-          add: {
-            content: "import type { StripeTypes } from '@hive/stripe-billing';",
-          },
-        },
```

Comment:
We are importing `StripeTypes` in [billing mappers](packages/services/api/src/modules/billing/module.graphql.mappers.ts) so we don't have to manually add it here